ui/packages/catalog-ui-search/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "fmt": "ace format -w",
     "prestart": "ace disable-idp",
-    "start": "ace start",
+    "start": "node --max_old_space_size=8192 ../../node_modules/@connexta/ace/bin.js start",
```

Comment:
Why do you need to give node this parameter for space size?

Reply:
We were running into out of memory issues.